routes/pages.js

```diff
@@ -15,8 +17,41 @@ router.get('/:siteName/pages', async function(req, res, next) {
     const IsomerFile = new File(access_token, siteName)
     const pageType = new PageType()
     IsomerFile.setFileType(pageType)
-    const pages = await IsomerFile.list()
-
+    let simplePages = await IsomerFile.list()
+    // After listing all simple pages, they're tagged for the frontend
+    simplePages = simplePages.map(simplePage => {
+      return {
+        ...simplePage,
+        type: 'simple-page'
+      }
+    })
+
+    const IsomerCollection = new Collection(access_token, siteName)
+    const collections = await IsomerCollection.list() //lists out all collections
+
+    /**
+     * This `reduce` function will:
+     * 1) Iterate through the collections
+     *  a) Lists all the collection pages with `CollectionPage.list()`
+     *  b) Map it to tag it with type `collection` [for frontend to know]
+     * 2) Concatenate it into the `accumulator`
+     * This then returns a flattened array of all collections pages from all collections (`allCollectionPages`)
+     */
+    const allCollectionPages = await Bluebird.reduce(collections, async (accumulator, collectionName) => {
+      const CollectionPage = new File(access_token, siteName)
+      const collectionPageType = new CollectionPageType(collectionName)
+      CollectionPage.setFileType(collectionPageType)
+      const collectionPages = await Bluebird.map(await CollectionPage.list(), (item) => {
```

Comment:
You don't need to use `await Bluebird.map` here. A simple map over the result of `await CollectionPage.list()` works because there is no async call within the map.

Reply:
The PR looks good otherwise

Reply:
ohhh yeah good point alright will fix that